WooCommerce/Classes/ViewModels/ProductDetailsCellViewModel.swift

```diff
@@ -21,6 +21,8 @@ struct ProductDetailsCellViewModel {
     ///
     private let product: Product?
 
+    private let productImageURL: URL?
+
```

Comment:
⚠️ I feel like there should be separate view model types at this point. But that's quite a refactoring. 

Would you consider renaming this to `preferredImageURL`? And maybe a comment about how it is used? 

Or I wonder if we should make `var imageURL` be intentionally initialized during `init` instead of it being a computed property. In this way, we only have one image URL property.

Reply:
yea, initializing `imageURL` in each initializer makes the most sense to me. `Product?` parameter in its initializers is actually only used for `imageURL`, I removed the private `Product?` here as well: https://github.com/woocommerce/woocommerce-ios/pull/3197/commits/b4315a2b690dbdf38dd9f3232484e3a2c86ce361

Reply:
Love it!